internal/testprocessor/runs.go

```diff
@@ -0,0 +1,169 @@
+package testprocessor
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconf "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	gconfig "github.com/gookit/config/v2"
+	"github.com/pkg/errors"
+	zlog "github.com/rs/zerolog/log"
+	"gopkg.in/yaml.v3"
+)
+
+const clickhouseDatabase = "clickhouse"
+
+type ImportRunsParams struct {
+	AwsAccessKeyID        string
+	AwsSecretAccessKey    string
+	AwsRegion             string
+	AwsQueryRunsTableName string
+
+	RunsBefore time.Time
+	RunsAfter  time.Time
+
+	OutputPath string
+}
+
+func (p *ImportRunsParams) Retrieve(_ context.Context) (aws.Credentials, error) {
+	return aws.Credentials{
+		AccessKeyID:     p.AwsAccessKeyID,
+		SecretAccessKey: p.AwsSecretAccessKey,
+		Source:          "local config",
+	}, nil
+}
+
+type RunsInput struct {
+	Runs []*Run `mapstructure:"runs" yaml:"runs"`
+}
+
+type Run struct {
+	Database    string        `mapstructure:"database" yaml:"database" dynamodbav:"Database"`
+	Version     string        `mapstructure:"version" yaml:"version" dynamodbav:"Version"`
+	Query       string        `mapstructure:"query" yaml:"query" dynamodbav:"Input"`
+	Timestamp   time.Time     `mapstructure:"timestamp" yaml:"timestamp" dynamodbav:"CreatedAt"`
+	TimeElapsed time.Duration `mapstructure:"-" yaml:"-" dynamodbav:"-"`
+}
+
+func (r *Run) CsvRow() []string {
```

Comment:
CSV is useful if we want to analyze runs in spreadsheets, and it doesn't seem to be a popular use case.

But users might want to write their own test scripts, and for such cases YAML seems more user-friendly (easy to edit).